app/javascript/menu/search.jsx

```diff
@@ -79,7 +79,7 @@ const MenuSearch = ({
     <SideNavItem className="padded menu-search">
       <Search
         size="sm"
-        placeHolderText={__('Find')}
+        placeholder={__('Find')}
```

Comment:
@kavyanekkalapu i thought this was going in a separate PR, as this change is not directly related to fix in this PR

Reply:
@h-kataria sorry about that, does it helps if i add this issue in description too?